ci/provisioning/post_provision_config.sh

```diff
@@ -48,7 +48,7 @@ if ! retry_cmd 2400 clush -B -S -l root -w "$NODESTRING" \
            REPO_FILE_URL=\"$REPO_FILE_URL\"
            ARTIFACTORY_URL=\"${ARTIFACTORY_URL:-}\"
            BRANCH_NAME=\"${BRANCH_NAME:-}\"
-           CHANGE_TARGET=\"${CHANGE_TARGET:-}\"
+           CHANGE_TARGET=\"${CHANGE_TARGET:-$BRANCH_NAME}\"
```

Comment:
Per the PR for the other branch, `$CHANGE_TARGET` should be considered immutable and you can do your this-or-that substitution where you need the either/or value.